src/main/java/org/opensearch/ubi/QueryRequest.java

```diff
@@ -79,10 +87,18 @@ public Map<String, String> getQueryAttributes() {
      * Gets the timestamp.
      * @return The timestamp.
      */
-    public long getTimestamp() {
+    public String getTimestamp() {
         return timestamp;
     }
 
+    /**
+     * Gets the application.
```

Comment:
sigh...    Not the hardest working comment ever!   In someways, I wonder if having a link in this Java to the spec would be enough in terms of javadocs?